server/src/main/java/server/BazelLanguageServer.java

```diff
@@ -36,7 +37,18 @@ public BazelLanguageServer() {
     logger.info(String.format("Starting up bazel language server with params:\n\"%s\"", params));
 
         initializeWorkspaceRoot(params);
-        Workspace.getInstance().initializeWorkspace();
+        try{
+            Workspace.getInstance().initializeWorkspace();
+            logger.info("workspace initialized");
+
+        } catch( BazelServerException e){
+            logger.info("workspace error");
```

Comment:
In the future, I think it'd be good if our server automatically requeried after the errors were fixed. But I think this is good for now.

Reply:
Yeah i agree. This is bare bone feature right now. 

It'd be nice to be able to:
- travel from the notification to the file that broke the bazel build
- Detect when the file is fixed and re-query. 
- Or at least have a button on the notification that would trigger a re-query
- Or have a command that triggers a re-query without having to restart the server.